src/main/java/de/zalando/ep/zalenium/proxy/DockeredSeleniumStarter.java

```diff
@@ -79,7 +79,7 @@
             "zalenium_https_proxy",
             "zalenium_no_proxy"
     };
-    
```

Comment:
Please revert changes to this file, it is just some spaces and tabs that changed.